src/loader/networks/sdex-chain.ts

```diff
@@ -1,20 +1,37 @@
-import type { DocumentNode } from 'graphql';
+import { ethers, ZeroAddress } from 'ethers';
+import { DocumentNode } from 'graphql';
+import { bignumber } from 'mathjs';
 
-import { SdexQuery, SdexQuery__factory } from 'artifacts/abis/types';
+import QueryContractABI from 'artifacts/abis/SdexQuery.json';
+import { Multicall, SdexQuery, SdexQuery__factory } from 'artifacts/abis/types';
 import { queryFromSubgraph } from 'loader/subgraph';
+import { LiquidityChangesResponse, PositionType } from 'typings/subgraph/liquidity';
+import {
+  aggregatePositions,
+  filterPositions,
+  parseAmbientTokensResult,
+  parseRangeTokensResult,
+  parseRewardResult,
+} from 'utils/aggregationUtils';
 import { loadGqlFromArtifacts } from 'utils/subgraph';
 
 import type { Chain } from './chain-config';
 import type { SdexChainConfig } from './types';
 
-const gglPools = loadGqlFromArtifacts('graphQueries/sdex/pools.graphql');
+const gqlPools = loadGqlFromArtifacts('graphQueries/sdex/pools.graphql');
+const gqlLiquidityChanges = loadGqlFromArtifacts('graphQueries/sdex/liqchanges.graphql');
 
 export class SdexChain {
   readonly query: SdexQuery;
-  // todo: add impact
+  readonly queryContract: ethers.Contract;
+  readonly multicall: Multicall;
```

Comment:
Please just use `this.context.multicall` in this class instead of reassigning it. Context is has shared utilities from parent class and made for this reason.

```suggestion
```